pom.xml

```diff
@@ -74,7 +74,7 @@
       <dependency>
         <groupId>org.json</groupId>
         <artifactId>json</artifactId>
-        <version>20240303</version>
+        <version>20241224</version>
```

Comment:
Better to use json-api plugin and exclude transitive dependencies that bring org.json

Reply:
Good point, maybe that should be added to enforcer rules for all of the existing wrapper plugins?